### PR TITLE
CICD Release Pipeline w/ Github Release Asset

### PR DESCRIPTION
**Pull request by cindydeng1998**
- Follow format of v0.0.0-rc0 for version tags
- Publish content of the dest/ folder as release asset on Github (https://github.com/Azure/iot-edge-config/releases)
- Releases are set as drafts until manual approval

### CODE REVIEW

File: vsts_ci/azure-pipelines.yml [hunk 1]
```diff
@@ -27,16 +27,27 @@ stages:
...
+      displayName: Create dest/ directory
+    - task: GitHubRelease@1
+      displayName: 'GitHub release (create)'
       inputs:
-        SourceFolder: '$(BUILD.REPOSITORY.LOCALPATH)/release'
-        TargetFolder: '$(Build.ArtifactStagingDirectory)'
-      displayName: "Copy files to build artifact staging directory"
-    
-    - task: PublishBuildArtifacts@1
+        gitHubConnection: 'github.com_cindydeng1998'
```
Comment (mhshami01) [inline vsts_ci/azure-pipelines.yml:43]: what if 'github.com_cindydeng1998' decides to move on to a different project?

**Review by mhshami01 — APPROVED** [review]

File: vsts_ci/azure-pipelines.yml [hunk 2]
```diff
@@ -27,16 +27,27 @@ stages:
...
+      - RaspberryPiUnitTests
+    condition: and(succeeded('LinuxUnitTests'), succeeded('RaspberryPiUnitTests')) 
     steps:
-    - task: CopyFiles@2
+    - script: |  
+        mkdir dest
+        cp src/azure-iot-edge-installer.sh dest/ 
+      displayName: Create dest/ directory
+    - task: GitHubRelease@1
+      displayName: 'GitHub release (create)'
```
Comment (marianan) [inline vsts_ci/azure-pipelines.yml:41]: >GitHub release (create) [](http://example.com/codeflow?start=20&length=23)

to distinguish this and next step, i suggest we append to the display name 'RC'

**Review by marianan — APPROVED** [review]
:shipit:

File: vsts_ci/azure-pipelines.yml (hunk 1 above)
Comment (cindydeng1998) [reply vsts_ci/azure-pipelines.yml:43]: We don't have a team Github account for now, and we can always change the service connection if 'github.com_cindydeng1998' decides to move on to a different project :)

**Pull request merged**